GoldRush.js

```diff
@@ -0,0 +1,320 @@
+
+class GoldRush extends Matrix{
+    constructor()
+    {
+        super(5,5)
```

Comment:
better have those `5` in a `const` outside (maybe in a `consts.js` file so you (or the player itself) could change that if wanted